App/signup.js

```diff
@@ -0,0 +1,49 @@
+function goOnTop(){
+  document.body.scrollTop = document.documentElement.scrollTop = 0;
+  document.getElementById("id").focus();
+}
+
+var playlistFromParse = JSON.parse(playlistsJSON);
+
+
+function unhidePlaylist(){
+
+    document.getElementById("playlist").style.visibility = "visible";
+    console.log(JSON.parse);
+    var currentPlaylist = new Playlist();
+
+    for(var i = 0; i <= playlistFromParse[0].songs[i].length; i++){
+      var song = new Song(playlistFromParse[0].songs[i]);
+      console.log(songs);
+
+      }
+    }
+
+
+function closePlaylist(){
+    document.getElementById("playlist").style.visibility = "hidden";
+}
+
+
+
+
+
+
+
+
+
+
+function Song(){
```

Comment:
Create for EACH class a separate file. Song->Song.js, Playlist->Playlist.js and son on.